pkg/autodiscovery/autoconfig_test.go

```diff
@@ -17,6 +17,7 @@ import (
 
 	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
 	"github.com/DataDog/datadog-agent/pkg/autodiscovery/listeners"
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery/providernames"
```

Comment:
```suggestion
	providernames "github.com/DataDog/datadog-agent/pkg/autodiscovery/providers/names"
```
like this it is inside the `autodiscovery/providers` path